cvrf/suse/suse.go

```diff
@@ -79,9 +80,16 @@ func (c Config) update(os string, urls []string) error {
 	for _, cvrfXml := range cvrfXmls {
 		var cv Cvrf
 		if len(cvrfXml) == 0 {
-			log.Printf("empty CVRF xml: %s", cv.Tracking.ID)
+			log.Println("empty CVRF xml")
 			continue
 		}
+
+		if !utf8.Valid(cvrfXml) {
+			log.Println("invalid UTF-8")
```

Comment:
nit: Should we print the ID for debugging purposes?

Reply:
I wanted to do that, but we don't have any ID here because it happens before parsing XML.

Reply:
I see.